review_practice/react-app/src/containers/RoomList/style.js

```diff
@@ -0,0 +1,159 @@
+import styled from 'styled-components';
+
+export const Main = styled.main`
+    && {
+        padding-top: 4rem;
+
+        .modal {
+            position: fixed;
+            top: 0;
+
+            width: 100%;
+            height: 100%;
+
+            background: rgba(0, 0, 0, 0.2);
+        }
+
+        .modal-dialog {
+            margin: 10rem 25%;
+        }
+    }
+
+    .RoomList-CountTitle {
+        margin: 1rem;
+        font-size: 2rem;
+        font-weight: bold;
+    }
+
+    @media only screen and (max-width: 800px) {
```

Comment:
mediar query 같은 경우도 color와 마찬가지로 파일로 만들어 관리하면 더 좋을것 같네요

Reply:
```suggestion
    ${isBigScreen} {
```
styled in JS 를 사용하는 만큼 
src/styles/mediaQuery.ts
```js
const bigScreenWIdth = 800;
export const isBigScreen = `@media only screen and (max-width:${bigScreenWidth})`;
```

Reply:
@crongo